src/ApiService/ApiService/Program.cs

```diff
@@ -36,27 +35,26 @@ public class LoggingMiddleware : IFunctionsWorkerMiddleware {
         public async Async.Task Invoke(FunctionContext context, FunctionExecutionDelegate next) {
             //https://learn.microsoft.com/en-us/azure/azure-monitor/app/custom-operations-tracking#applicationinsights-operations-vs-systemdiagnosticsactivity
             using var activity = OneFuzzLogger.Activity;
-            _ = activity.Start();
-            string correlationId = Guid.NewGuid().ToString();
-
-            if (await context.GetHttpRequestDataAsync() is HttpRequestData requestData) {
```

Comment:
why did you remove getting Correlation ID from the http header ? 

request comes it -> we'd get Correlation ID from the request, and we can use that in the rest of the execution flow.

Reply:
If execution flow starts with CLI, how does CLI set the initial correlation ID ?